etc/profile-m-z/rtv-addons.inc

```diff
@@ -0,0 +1,23 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+
+# You can configure rtv to open different type of links
+# in external applications. Configuration here:
+# https://github.com/michael-lazar/rtv#viewing-media-links
+# This include is meant to facilitate that configuration
+# with the use of a .local file.
+
+ignore nosound
```

Comment:
machine-id is also known to break audio, please double-check if that needs to be ignored too

Reply:
mpv seams to work, i'm on 18.04 though...
The sound is for mpv with youtube-dl, with other players, you'll need a direct link, that's rarer.

Reply:
Great, let's go without touching machine-id in that case.